extensions/Worker.Extensions.CosmosDB/src/Utilities.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.CosmosDB
+{
+    internal class Utilities
```

Comment:
nit - could this be named `CosmosDBUtilities` ? Or is the intention for this class to be used for utilities for purposes outside of that scope?

Reply:
It's in the Cosmos project so that's specific enough for CosmosDB, especially as we have the namespace here to scope it